src/server/resolve.rs

```diff
@@ -82,7 +81,14 @@ where
 
     fn get_address(&self, store_id: u64) -> Result<String> {
         let pd_client = Arc::clone(&self.pd_client);
-        let mut s = box_try!(pd_client.get_store(store_id));
+        let mut s = match pd_client.get_store(store_id) {
+            Ok(s) => s,
+            Err(pd_client::Error::StoreTombstone(_)) => {
+                RESOLVE_STORE_COUNTER_STATIC.tombstone.inc();
+                return Err(box_err!("store {} has been removed", store_id));
```

Comment:
How about comment here about where the error is handled so that other guys won't make mistakes again?